packages/openneuro-components/src/search-page/FiltersBlockContainer.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+import { FiltersBlock } from './FiltersBlock'
+
+export const FiltersBlockContainer = () => {
+  return <FiltersBlock />
+}
```

Comment:
This could either be renamed to `FiltersBlockContainerExample`, or, since this is just static data, we could use `<FiltersBlock/>` as-is in `renderSearchFacets`.